src/welcome-day-1.md

```diff
@@ -18,10 +18,10 @@ Please remind the students that:
 * They should ask questions when they get them, don't save them to the end.
 * The class is meant to be interactive and discussions are very much encouraged!
   * As an instructor, you should try to keep the discussions relevant, i.e.,
-    keep the related to how Rust does things vs some other language. It can be
+    keep it related to how Rust does things vs some other language. It can be
```

Comment:
I'm not a native speaker (as all the grammar fixing PRs have revelased :smile: ) but should this not be "them" since it refers back to "the discussions", which is plural?

Reply:
Good catch, thank you!